robottelo/constants/__init__.py

```diff
@@ -367,9 +367,6 @@ class Colored(Box):
     'rhel-8-for-x86_64-appstream-rpms',
 )
 
-INSTALL_RHEL7_STEPS = 'yum -y install satellite'
-INSTALL_RHEL8_STEPS = 'dnf -y module enable satellite:el8 && dnf -y install satellite'
-
```

Comment:
This might be required in 6.11 as the test should be parametrized for el7 and el8, right?

Reply:
Updated. Not removing now